src/components/atom/button/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactComponent as AddIcon } from "../../../assets/img/addIcon.svg";
+import "./styles.scss";
+
+type ButtonProps = {
+  text: string;
+  hasAddIcon?: boolean;
+  onClick: () => void;
+  type: HTMLButtonElement["type"];
+  disabled?: boolean;
+  styles: string;
+};
+
+const Button = ({ text, hasAddIcon, onClick, type, disabled, styles, ...rest }: ButtonProps) => {
```

Comment:
type should be default button

Reply:
Default type should be a button - to prevent submit behavior.